src/front/components/map/background.vue

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+    import * as atlas from "azure-maps-control";
+    import "azure-maps-control/dist/atlas.min.css";
+import { LanguageServiceMode } from "typescript";
+
+    export default {
+        mounted() {this.initializeMap()}, 
+        methods: {
+            initializeMap() {
+                const map = new atlas.Map("map", {
+                    view: "Auto",
+                    language: "fr-FR",
+                    center: [-0.604945545248392, 44.806516403595744],
+                    zoom: 10,
+                    authOptions: {
+                        authType: "subscriptionKey" as atlas.AuthenticationType,
+                        subscriptionKey: "76c6F3ZTQ9iJDRo20oAU-D5YtyP7wkG-s1BRffLLgbc"
```

Comment:
Il faudrait stocker la clé dans un fichier d'environnement, voir [Microsoft Learn - Azure Maps - Secure a daemon application](https://learn.microsoft.com/en-us/azure/azure-maps/how-to-secure-daemon-app)

Reply:
Pour les variables d'environnement, il faut en fait les référencer dans le fichier `nuxt.config.ts` et y accéder avec la runTimeConfig, voir [Nuxt - Runtime Config](https://nuxt.com/docs/guide/going-further/runtime-config).